articles/connections/references/apple-native/guides/add-siwa-to-web-app.md

```diff
@@ -0,0 +1,243 @@
+
+## Prerequisites
+
+Before you add support for SIWA to your web app, you'll need:
+
+* An [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/)).
+* A domain (such as <YOUR CUSTOM DOMAIN>.com) to point to and an internet-accessible server where you will run the app that responds on behalf of this domain. You will also need to configure this server with a TLS certificate (Apple won't accept unsecured HTTP connections). To use the Email Relay Service, you will also need to configure your domain with Sender Policy Framework (SPF) DNS TXT records.
+
+::: note
+You can set this up using [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04), [Freenom](https://freenom.com/), or [Let's Encrypt](https://letsencrypt.org/).
+:::
+
+* A [Custom Domain](/custom-domains) set up on your Auth0 tenant for domain verification with Apple. Custom domains are not strictly required to use SIWA authorization. Domain validation is required for sending emails to private Apple addresses in native and web apps. 
+
+## Register your web app in your Apple Developer account
+
+When setting up your application with your Apple Developer account, make sure you save the following IDs and keys for the application connection settings in the Auth0 Dashboard:
+
+* Client ID (the Service ID)
+* App ID
+
+1. In your Apple Developer account, go to the [Certificates, IDs, & Profiles](https://developer.apple.com/account/resources/certificates/list) section and choose the [Identifiers](https://developer.apple.com/account/resources/identifiers/list/serviceId) from the left menu.
+
+3. Click **Register a Service ID** button.
+
+4. Choose **Service IDs** as the identifier type, and click **Continue**. 
+
+5. Enter a description for your new App ID and a Service ID. For the latter, Apple recommends using a reverse-domain name style string (for example, `com.<YOUR CUSTOM DOMAIN>.appid`). 
+
+6. Scroll down and check the **Sign In with Apple** feature. You won't have to use the **Edit** here. You will configure this feature later.
+
+7. Leave the other options with their default values and click **Continue**. When you click on this button, Apple displays a summary of the options you just configured. If everything is correct, click **Register**. Apple redirects you to the **Certificates, Identifiers & Profiles** subsection again, this time listing your new App ID. 
+
+8. Click **Download** for Apple to send you a file called `apple-developer-domain-association.txt`. You will have to use the contents of this file soon. Also, keep the page open because you will have to use the **Verify** button later.
```

Comment:
Download button is only available in the Redirect URI/domain validation dialog.  Not in the **Certificates, Identifiers, and Profiles** home.